examples/patterns/navigation/subnav.html

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: Navigation / Sub Navigation
+category: _patterns
+---
+<header id="navigation" class="p-navigation">
+  <div class="p-navigation__banner">
+    <div class="p-navigation__logo">
+      <a class="p-navigation__link" href="#">
+        <img class="p-navigation__image" src="https://assets.ubuntu.com/v1/d96d86b5-vanilla_black-orange_hex.svg" alt="" width="95" />
+      </a>
+    </div>
+    <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+    <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+  </div>
+  <nav class="p-navigation__nav">
+    <form class="p-search-box" action="/search">
+      <input type="search" class="p-search-box__input" name="q" placeholder="Search" required="">
+      <button type="reset" class="p-search-box__reset" alt="reset"><i class="p-icon--close"></i></button>
+      <button type="submit" class="p-search-box__button" alt="search"><i class="p-icon--search"></i></button>
+    </form>
+    <span class="u-off-screen">
+      <a href="#main-content">Jump to main content</a>
+    </span>
+    <ul class="p-navigation__links" role="menu">
+      <li class="p-navigation__link p-subnav" role="menuitem" id="link-1">
+        <a href="#link-1-menu" class="p-subnav__toggle">Link 1</i>
+        </a>
+        <ul class="p-subnav__items" id="link-1-menu" aria-hidden="true" style="display: none;">
```

Comment:
Please put the styling on the class

Reply:
@anthonydillon I misunderstood your previous suggestion - all done with the cascade now and the JS just switches aria attributes.